### ISSUE TITLE
conda environment of roary is not resolving

### ISSUE THREAD

**Issue by klamkiew** [hoelzer-lab/ribap#39]
see title, I am working on it ;)

**Comment by hoelzer** [comment]
Argh, what's going on with all theses condo envs ^^ damn it

**Comment by klamkiew** [comment]
Funnily enough, I can resolve the env with mamba... not sure why conda is having issues here.... maybe we can plug in mamba into NF, even though it is experimentally? @hoelzer ever done this before? :D

**Comment by hoelzer** [comment]
Yes Mamba profiles work, Marie implemented one in... Clean I think? Or rnaflow? We can ask her and c&p ;)

**Comment by hoelzer** [comment]
And strange the roary condo thing... Good we build also a container 3y ago ^^

**Comment by klamkiew** [comment]
Once I find some time, I'll try to dig deeper into that issue; for now, maybe the mamba fix is already sufficient. But we also had issues with the prokka env, so I think there might be some underlying conda config problem :/

**Comment by klamkiew** [comment]
Alright; I set `useMamba = true` in the conda scope of our nextflow config. Now, one has to have nextflow and mamba, if one wants to use conda, but this isn't much of a hussle with conda anyways.
Closing this for now -- I still have to check whether a switch between conda and mamba is possible (e.g. a profile mamba, that looks like conda but uses mamba)

**Comment by hoelzer** [comment]
great thx! 

Honestly, for me it's also totally fine if the user needs nextflow +

* mamba
or
* docker/singularity

and conda is not natively supported. Of course, if we can have a profile that looks like conda but just installs and uses mamba: fine ;) But maybe not worth the hazzle
